### PR TITLE
travis: Use beaver to build the project

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**
* submodules/beaver ()...v0.2.0+4(138f9e7) (1 commits)
  > dlang: Don't try to convert D2 only projects

### CODE REVIEW

**Comment by leandro-lucarella-sociomantic** [comment]
Blocked because we need to wait for the next release of beaver to support D2-only projects.

File: .gitmodules [hunk 1]
```diff
@@ -19,3 +19,6 @@
 [submodule "tinyendian"]
 	path = submodules/tinyendian
 	url = https://github.com/kiith-sa/tinyendian
+[submodule "beaver"]
+	path = submodules/beaver
+	url = https://github.com/leandro-lucarella-sociomantic/beaver.git
```
Comment (leandro-lucarella-sociomantic) [inline .gitmodules]: I need to change this for the final version.

**Comment by leandro-lucarella-sociomantic** [comment]
This should be ready to merge, the only *problem* is there is no release yet with the needed changes, but since this tools is still under some intense development, I think it's not an issue in practice.

File: src/release/gitHelper.d [hunk 2]
```diff
@@ -102,7 +102,8 @@ string getRemote ( string upstream )
     auto remotes = cmd("git remote -v")
                       .splitter!(a=>a == '\n')
                       .uniq()
-                      .filter!(a=>a.canFind("github.com:" ~ upstream))
+                      .filter!(a=>a.canFind("github.com:" ~ upstream) ||
+                                  a.canFind("github.com/" ~ upstream))
```
Comment (mihails-strasuns-sociomantic) [inline src/release/gitHelper.d]: Wrong base?

**Pull request merged**